### PR TITLE
Parsing Rules Filter Fix Round 2

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-7462](https://jira-hq.paloaltonetworks.local/browse/CIAC-7462)

## Description
This PR adds filter for parsing rules in the following packs:
1. Workday
2. ManageEngine - AD Manager
3. Mimecast

### CODE REVIEW

File: Packs/ManageEngine-ADManager/README.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+
+<~XSIAM>
 ## Configuration on Server Side
 
 Steps to enable Syslog Logging in ADManager Plus:
 1. Log in to ADManager Plus.
 2. Go to **Admin** > **Personlize** > **Integration**
 3. Click the **Syslog Server** option.
-4. Enter the details including Syslog Server Name, Port Number and Port Protocol. Also, choose the syslog standard and specify the data format needed for your XSIAM parser.
+4. Enter the details the on the [*Syslog Settings*](https://www.manageengine.com/products/ad-manager/images/admanager-plus-being-integrated-with-syslog.png) form, including the **Syslog Server Name**, **Port Number** and **Port Protocol**.  In addition, select the [*RFC 5424*](https://datatracker.ietf.org/doc/html/rfc5424) **Syslog Standard** (recommended), and specify the requested data format for your Cortex XSIAM parser.
```
Comment (ShirleyDenkberg) [inline Packs/ManageEngine-ADManager/README.md:12]:
```suggestion
4. Enter the details on the [*Syslog Settings*](https://www.manageengine.com/products/ad-manager/images/admanager-plus-being-integrated-with-syslog.png) form, including the **Syslog Server Name**, **Port Number** and **Port Protocol**.  In addition, select the [*RFC 5424*](https://datatracker.ietf.org/doc/html/rfc5424) **Syslog Standard** (recommended), and specify the requested data format for your Cortex XSIAM parser. 
```

File: Packs/ManageEngine-ADManager/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Parsing Rules
+
+##### ManageEngine-ADManager Parsing Rule
+
+- Added support for timestamps which are compatible with RFC 5424. 
+- Updated the parsing rules metadata ID & name for overall convention compatibility.
```
Comment (ShirleyDenkberg) [inline Packs/ManageEngine-ADManager/ReleaseNotes/1_0_1.md:7]:
```suggestion
- Updated the parsing rules metadata ID and name for overall convention compatibility.
```

File: Packs/MicrosoftExchangeServer/ReleaseNotes/1_0_6.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### MicrosoftExchangeServer Parsing Rule
+
+- Added a filter in the parsing rule to enhance its logic.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeServer/ReleaseNotes/1_0_6.md:6]:
```suggestion
Added a filter in the parsing rule to enhance its logic.
```

File: Packs/Workday/ReleaseNotes/1_3_6.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### Workday Parsing Rule
+
+- Added a filter in the parsing rule to enhance its logic.
```
Comment (ShirleyDenkberg) [inline Packs/Workday/ReleaseNotes/1_3_6.md:6]:
```suggestion
Added a filter in the parsing rule to enhance its logic.
```

**Comment by ShirleyDenkberg** [comment]
@nkanon @cweltPA Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**